server/neptune/gateway/event/pull_request_review_handler.go

```diff
@@ -47,7 +59,16 @@ func (p *PullRequestReviewWorkerProxy) handle(ctx context.Context, event PullReq
 	if event.State != Approved {
 		return nil
 	}
+	if err := p.handleLegacyMode(ctx, request, event); err != nil {
+		p.Logger.ErrorContext(ctx, err.Error())
+	}
+	if err := p.handlePlatformMode(ctx, event); err != nil {
+		return errors.Wrap(err, "handling platform mode")
+	}
```

Comment:
why not do this up a level? ie. just schedule these two separately as part of the async scheduler so we get all the error handling built in and they execute in parallel instead of sequentially

Reply:
sure, moved it up a level